index.js

```diff
@@ -254,7 +262,8 @@ class Analytics {
       return Promise.resolve()
     }
 
-    const items = this.queue.splice(0, this.flushAt)
+    this.queueBytes = 0
+    const items = this.queue.splice(0, this.queue.length)
```

Comment:
I changed this away from `this.flushAt`, because it _should_ be functionally-equivalent under the current implementation, and also because it could potentially lead to confusing bugs where calls to `flush()` might actually leave items on the queue (which seems like it would _never_ be desirable)